src/components/media/controls/videoControls.tsx

```diff
@@ -60,6 +60,8 @@ export function PictureInPictureButton(props: PictureInPictureButtonProps) {
 export function FullscreenButton(props: FullscreenButtonProps) {
   const [isFullscreen, setIsFullscreen] = useState(!!document.fullscreenElement)
 
+  const videoAndControlsContainer = props.element
```

Comment:
Should we call it `videoContainer` instead? You passed `videoContainerRef.current` to `props.element`. I think the names should be similar.
```suggestion
  const videoContainer = props.element
```